api-reference/10 UI Components/dxForm/1 Configuration/aiIntegration.md

```diff
@@ -5,7 +5,92 @@ default: undefined
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Binds the AI service to the Form.
+
+---
+The Form component ships with the **Smart Paste** functionality. To activate this capability, specify:
+
+- [AIIntegration](/api-reference/40%20Common%20Types/AIIntegration '/Documentation/ApiReference/Common_Types/AIIntegration/') class
+- *'smartPaste'* button [name](/Documentation/ApiReference/UI_Components/dxForm/Types/#FormPredefinedButtonItem) (or call the [smartPaste(text)](/Documentation/ApiReference/UI_Components/dxForm/Methods/#smartPastetext) method)
```

Comment:
I want the list to be more consistent. Right now, you list a class in the first item (you must assign that class to the property, as I understand). The second item starts with what I believe is a property name. 

I would be better if both items start with the property name.

```suggestion
- *`aiIntegration`* - accepts an [AIIntegration](/api-reference/40%20Common%20Types/AIIntegration '/Documentation/ApiReference/Common_Types/AIIntegration/') object that holds AI Service settings.
- *'smartPaste'* - specifies a button [name](/Documentation/ApiReference/UI_Components/dxForm/Types/#FormPredefinedButtonItem). (You can also call the [smartPaste(text)](/Documentation/ApiReference/UI_Components/dxForm/Methods/#smartPastetext) method.)
```